### ISSUE TITLE
Add better alt texts for all the images in the benchmarking rst file

### ISSUE THREAD

**Issue by qh681248** [gchq/coreax#985]
### What's the new feature?

Alternative text should be more descriptive. Add better alt texts for all the images in the benchmarking rst file.

### What value does this add?

Improves accessibility of documentation

**Comment by gw265981** [comment]
Blocked by #977.
